spec/std/callstack_spec.cr

```diff
@@ -1,68 +1,58 @@
 require "./spec_helper"
 
-describe "Backtrace" do
-  it "prints file line:colunm" do
-    with_tempfile("compiler_spec_output") do |path|
-      sample = datapath("backtrace_sample")
+private def compile_and_run_file(source_file)
+  with_tempfile("executable_file") do |executable_file|
+    puts executable_file
```

Comment:
This debug code shouldn't have been merged.

Reply:
Ooops.